ml-agents/mlagents/trainers/action_info.py

```diff
@@ -8,10 +8,9 @@
 class ActionInfo(NamedTuple):
```

Comment:
I know this is not part of this PR, but can you add a comment for each of the entries on the ActionInfo tuple ?